Formula/openldap.rb

```diff
@@ -63,13 +59,12 @@ def install
       --enable-translucent
       --enable-unique
       --enable-valsort
+      --without-systemd
     ]
 
-    if OS.linux?
-      args << "--without-systemd"
-
+    if OS.linux? || MacOS.version >= :ventura
       # Disable manpage generation, because it requires groff which has a huge
-      # dependency tree on Linux
+      # dependency tree on Linux and isn't included on macOS since Ventura.
       inreplace "Makefile.in" do |s|
         s.change_make_var! "SUBDIRS", "include libraries clients servers"
```

Comment:
```suggestion
        subdirs = s.get_make_var("SUBDIRS")
                   .split
                   .reject { |subdir| subdir["doc"] }
        s.change_make_var! "SUBDIRS", subdirs.join(" ")
```
This way we don't accidentally drop any new entries in `SUBDIRS`.